### DISCUSSION TITLE
link generation photo sharing

### DISCUSSION THREAD

**Discussion by doreks** [photoprism/photoprism#1040] [Q&A] [answered]
sorry for my question, but it''s not clear to me. 
To be able to share photos with third parties, friends, family, is it only possible to share with them if you have previously enabled nextcloud webdav?
I mean, if I don't have a nextcloud server, wouldn't it be possible to generate a "temporary" link to share the photos/albums within mydomain.com?

thank you very much for the help and very good work.

**Comment by alexisLefebvre** [comment]
No, you don't need WebDav to share photos. Try on the demo: https://demo.photoprism.org/albums/aqokbmd3cuzeeohu/flowers

**Comment by doreks** [comment]
I have noticed that, but in my installed version, which is the latest version of docker, I only have the option to share via webdav, I do not know if I am missing something.

**Comment by lastzero** [comment, marked as answer]
See docs: https://docs.photoprism.org/user-guide/share/share/

**Reply by doreks** [reply]
OMG!!! I can't believe it, I finally have it!!! thanks , thanks so much!
_I think it threw me off that I was looking for a link per individual photo._ 
i have notice that once i shared the folder or album im shared all their contained pictures, but is there any way to share only for instance, one picture without share all the album?

and other question, which is the diff between this two values?, i mean, i can understand the first one but what really do the second value?


PHOTOPRISM_SITE_URL | --site-url URL | public site URL (default: "http://localhost:2342/")

PHOTOPRISM_SITE_PREVIEW | --site-preview URL | public preview image URL



thanks!

**Reply by graciousgrey** [reply]
Sharing single photos is planned https://github.com/photoprism/photoprism/issues/466.

PHOTOPRISM_SITE_URL is for the url, PHOTOPRISM_SITE_PREVIEW is for the preview image
